src/extension.ts

```diff
@@ -83,6 +84,7 @@ function filterObject<V>(obj: { [key: string]: V }, callback: (entry: [string, V
 function updateConfiguration() {
     appendNewline = vscode.workspace.getConfiguration(rootSection).get("appendNewline")!;
     enableDiagnostics = vscode.workspace.getConfiguration(rootSection).get("enableDiagnostics")!;
+    showInternalNameHints = vscode.workspace.getConfiguration(rootSection).get("showInternalNameHints")!;
```

Comment:
Not sure how this would be changed with the new key, I don't have an editor handy.